src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1066,8 +1066,6 @@ void setOptimalIndex(int newOptimalIdx)
          */
         void rtcpTransform(ByteArrayBuffer baf)
         {
-            SeenFrame maxSentFrame = this.maxSentFrame;
-
```

Comment:
I think the local variable is there intentionally -- this.maxSentFrame can be modified by other threads.